### DISCUSSION TITLE
Running bipes  block code without any IOT device connected to it?

### DISCUSSION THREAD

**Discussion by RiteshF7** [BIPES/BIPES#204] [Q&A]
Hello everyone,
I want to run block code without connecting any hardware device connected to my laptop.
I want to run pure micropython code but on bipes without connecting any external device.
Help of any kind will be appreciated.
Thanks for your time.

**Comment by jetannenbaum** [comment]
I think you are looking for Scratch, not BIPES.  Scratch will allow you to use blocks to generate code.  BIPES expects you to connect to a device.
